src/api.ts

```diff
@@ -24,12 +24,12 @@ export interface MagicLinkResult {
 
 export async function createMagicLink(
   email: string,
-  options?: HttpOptions,
```

Comment:
This file is why I hate trailing commas!

Reply:
I love them, our prettier settings seem out of sync now though. Not very imporant to fix atm though.